openpype/pipeline/workfile/build_workfile.py

```diff
@@ -651,13 +660,15 @@ def _collect_last_version_repres(self, asset_docs):
         ```
         """
 
+        from openpype.pipeline.context_tools import get_current_project_name
```

Comment:
Any reason why this import can't be at the top of the file?

Reply:
Causing cross import bug.